action.yml

```diff
@@ -75,7 +74,7 @@ runs:
     - uses: codex-team/action-nodejs-package-info@v1
       if: inputs.npm-version == ''
       id: package
-    - uses: google-github-actions/release-please-action@v3
+    - uses: google-github-actions/release-please-action@v4
```

Comment:
This repo is marked as deprecated. If we're updating then use `googleapis/release-please-action` or revert it back to v3